.unit-test/gameboard-tests/services/endOfGame-test.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+    'use strict';
+    describe('Test endOfGame', function () {
+        var state,
+            timeout,
+            endedgame;
+
+        beforeEach(function(){
+            module('ui.router');
+            module('Controllers.MyModule');
+            module('Services.MyModule', function(){
+            });
+
+            inject(function($injector){
+                state= $injector.get('$state');
+                timeout = $injector.get('$timeout');
+                endedgame = $injector.get('EndOfGame')
+            });
+        });
+
+        it('Checks that the state changes to draw when game is drawn.', function(){
+            endedgame.outcome ='Draw';
+            endedgame.gameEnded();
+            timeout.flush();
+            state.current.url.should.equal('/draw');
```

Comment:
Better to inject a mock state service with a go method - then check the go method has been called with correct statename - decouples your test from the $state setup in config.